code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm

```diff
@@ -108,13 +117,45 @@
 	icon_state = "boltgun_wood"
 	item_suffix  = "_wood"
 	force = 23
-	recoil_buildup = 1.7 // however, since it's not the excel mosin, it's not as good at recoil control, but it doesn't matter since >bolt
+	recoil_buildup = 2.8 // however, since it's not the excel mosin, it's not as good at recoil control, but it doesn't matter since >bolt
 	matter = list(MATERIAL_STEEL = 20, MATERIAL_WOOD = 10)
 	wielded_item_state = "_doble_wood"
 	spawn_blacklisted = FALSE
 	gun_parts = list(/obj/item/stack/material/steel = 16)
 	sawn = /obj/item/gun/projectile/boltgun/obrez/serbian
 
+/obj/item/gun/projectile/boltgun/fs
+	name = "FS BR .20 \"Arisaka\""
+	desc = "Weapon for hunting, or endless coastal warfare. \
+			Replica of an ancient bolt action known for its easy maintenance and low price."
+	icon_state = "arisaka"
+	item_suffix  = "_arisaka"
+	force = WEAPON_FORCE_DANGEROUS // weaker than novakovic, but with a bayonet installed it will be slightly stronger
+	armor_penetration = ARMOR_PEN_GRAZING
+	caliber = CAL_SRIFLE
+	damage_multiplier = 1.6
+	penetration_multiplier = 1.9
+	recoil_buildup = 3.2
+	max_shells = 6
+	magazine_type = /obj/item/ammo_magazine/srifle
+	matter = list(MATERIAL_STEEL = 15, MATERIAL_WOOD = 10, MATERIAL_PLASTEEL = 10)
+	wielded_item_state = "_doble_arisaka"
+	sharp = FALSE
+	spawn_blacklisted = FALSE
+	saw_off = FALSE
+	gun_parts = list(/obj/item/stack/material/steel = 16)
+
+/obj/item/gun/projectile/boltgun/fs/ih
+	name = "FS BR .20 \"Tosshin\""
+	icon_state = "arisaka_ih"
+	item_suffix  = "_arisaka_ih"
+	damage_multiplier = 1.6
+	penetration_multiplier = 1.7
+	recoil_buildup = 2.7 //trades penetration for recoil
+	matter = list(MATERIAL_STEEL = 25, MATERIAL_PLASTIC = 15)
+	wielded_item_state = "_doble_arisaka_ih"
+	spawn_blacklisted = TRUE
```

Comment:
Still don't understand why there are 2 FS boltguns with slightly different stats.

Reply:
One's for the armory, the other is a more specialised novak.
We have a lot of very similar weapons as is, such as almost identical smgs, and then there are the AKs. These at least provide a niché: mid- to low-tier weapons outperformed by most of their peers, but more capable of countering heavy armor and shields.

Reply:
> One's for the armory, the other is a more specialised novak.

Imo there is no point in more specialized Novak, those are the same apart of itsy bitsy stat difference that no one will ever notice.

> A simpler explanation is, lore.

What do you mean?

Reply:
> no one will ever notice.

I was probably a little reductive. Unlike the novak, the Arisaka uses .20, and with its stats this makes it significantly more effective against armor and shields, at the cost of damage and reliability.
This makes the Arisaka a poor man's lategame rifle. (It also lacks a bayonet, and this alongside the smaller magazine makes it less effective in close combat, encouraging a different playstyle - long range skirmishing)

The IH version, on the other hand is a simpler to handle underpowered version of the Arisaka, acting as a reliable and cheap way to combat armor for IH forces. Definitely more powerful against mercenaries than a Sol, without being much more expensive.

> What do you mean?

The Arisaka is a boltgun created by FS for mass production and sale, based on the old Japanese rifle, but modified to perform well against high-tech armor, combat RIGs or even lighter mechs.

The Tosshin is a downtuned version in use by Ironhammer forces, for on-board combat against heavily armored infantry.

Being an Ironhammer rifle, the Tosshin is blacklisted from spawning, while the Arisaka serves its purpose of being a midtier anti-armor gun through its presence as a moderately common gun disk. I also have further plans with it, but those are of no significance for the time being.

Reply:
The lore explanation is fine but I am really struggling to find an in game reason for the other gun, IH *really* does not need more guns, especially a boltgun that was made as a cheap weapon for vagabonds, but okay IH does not have any high AP single shot weapons, so that can pass, but the other variant that is a "single shot, good AP, good damage, mass produced boltgun for cheap"... do you know what else is a "single shot, good AP, good damage, mass produced boltgun for cheap"? The boltgun we already have! They even look the same. The caliber and number changes aren't enough to justify a whole new gun filling the same niche. It doesn't even fill that niche well as a "poor man's mass produced rifle" has ammo that is nowhere to be found. Calling on AKs is meaningless as they are the reason the gun ban is even in effect.
<details>
  <summary>
    Overall
  </summary>

One and possibly more items changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

</details>